src/__tests__/Calculator.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
- [ ] I would not recommend disabling the linter. Instead, you can optimize your code and follow best practices to avoid any linter issues.

Reply:
linters has been enable